### ISSUE TITLE
This package doesn't seem to be accessibility compliant

### ISSUE THREAD

**Issue by abhishek-msft** [microsoft/powerbi-client-react#40]
The component doesn't seem to be accessibility compliant:

![image](https://user-images.githubusercontent.com/66029786/113548831-d9ef1e00-960d-11eb-8b91-7370c6fc2970.png)

Especially, the iframe seems to be missing some accessibility attribute that is generated to display the powerbi :
![image](https://user-images.githubusercontent.com/66029786/113548900-03a84500-960e-11eb-8f98-829b11d61d89.png)

**Comment by abmis-mi** [comment]
Hi, Any update on this?

**Comment by fxvarga** [comment]
> Hi, Any update on this?

Is this an issue with the package or with the powerbi client package?

**Comment by abmis-mi** [comment]
This was with the powerbi client package.

**Comment by KotanaSai21** [comment]
Thank You for pointing this out. Fixed the accessibility issues with the demo app. 

To add the title attribute to your component, use the `setComponentTitle` API.

Refer to [Accessibility for your Power BI embedded analytics embedded solution | Microsoft Learn](https://learn.microsoft.com/javascript/api/overview/powerbi/accessibility#title-attribute) to make embedded reports more accessibility compliant.

Feel free to reach out for any other queries.
